modular_skyrat/modules/microfusion/code/microfusion_cell_attatchments.dm

```diff
@@ -0,0 +1,131 @@
+/*
```

Comment:
change file name to `modular_skyrat/modules/microfusion/code/microfusion_cell_attachments.dm`